qiime/split_libraries_lea_seq.py

```diff
@@ -194,6 +201,8 @@ def get_LEA_seq_consensus_seqs(fwd_read_f, rev_read_f,
                                  primer_mismatch_count,
                                  seq_too_short_count,
                                  total_seqs_kept)
+    if consensus_seq_lookup is None:
```

Comment:
Strange, from reading the code I can't see how `consensus_seq_lookup` can be `None` because it is created by `get_consensus_seqs_lookup`, which will always return a `defaultdict`. I'm probably missing something obvious -- can you point me to the line of code where `consensus_seq_lookup` can be set to `None`?